Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:14.3.0-alpine as builder
+
+WORKDIR /opt/thirdparty-api-adapter
+
+RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool autoconf automake \
+    && cd $(npm root -g)/npm \
+    && npm config set unsafe-perm true \
+    && npm install -g node-gyp 
+
+# check in .dockerignore what is skipped during copy
+COPY . .
```

Comment:
It's best to just copy the `package.json` and `package-lock.json` file here before running the `npm ci`, otherwise docker will rerun the `npm ci` every time you change files in `src`.

I'm happy to leave this for now, since it's not too important

Reply:
Ok Fixed now.